### ISSUE TITLE
Data loading is slow compared to instant-ngp

### ISSUE THREAD

**Issue by itaidotan** [nerfstudio-project/nerfstudio#1688]
Hi, I noticed that compared to instant-ngp the data loading time is much, much longer in Nerfstudio. I assume that the reason is the usage in Cuda which accelerates the loading, but I’m not sure.

If this is the case, is there intention to add Cuda data loader to the repo and allow users to use it to get faster training?
If this isn’t the case, does anyone know how to accelerate the loading time?

Thanks I’m advance!

**Comment by tancik** [comment]
Yes, there is a data processing bottleneck - https://github.com/nerfstudio-project/nerfstudio/issues/1638

**Comment by elenacliu** [comment]
Does anyone have any ideas?
